lib/components/TreeNode.js

```diff
@@ -34,7 +38,7 @@ class TreeNode extends React.Component {
         onClick={this.onClick}
         >
         <div className='list-item'>
-          <span className={thisItemClassName}>{this.props.text} ({this.props.nodes.length})</span>
+          <span className={thisItemClassName}>{this.props.text}{childrenCount}</span>
```

Comment:
Looks like this tweaked the build.
https://travis-ci.org/reergymerej/todo/builds/209111763#L419